sdk/storage/Azure.Storage.Files.DataLake/api/Azure.Storage.Files.DataLake.netstandard2.0.cs

```diff
@@ -784,3 +784,12 @@ public sealed partial class DataLakeSasQueryParameters : Azure.Storage.Sas.SasQu
         public override string ToString() { throw null; }
     }
 }
+namespace Microsoft.Extensions.Azure
+{
+    public static partial class DataLakeClientBuilderExtensions
+    {
+        public static Azure.Core.Extensions.IAzureClientBuilder<Azure.Storage.Files.DataLake.DataLakeServiceClient, Azure.Storage.Files.DataLake.DataLakeClientOptions> AddBlobServiceClient<TBuilder>(this TBuilder builder, System.Uri serviceUri) where TBuilder : Azure.Core.Extensions.IAzureClientFactoryBuilderWithCredential { throw null; }
```

Comment:
This should be `AddDataLakeServiceClient`

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/15331